source/tutorials/security/trusty-enabling/trusty-enabling.rst

```diff
@@ -0,0 +1,73 @@
+.. _trusty-enabling:
+
+How to Enable or Disable Trusty for Debugging
+#############################################
+
+`Trusty <https://01.org/trusty-ia>`_ is a mandatory component since Android Oreo dessert onwards. It provides the hardware backed security services such as the keymaster, gatekeeper, crypto etc, which are required for applications such as data storage encryption etc.
```

Comment:
Suggest replacing "etc" with "and so forth" plus minor rewrite:
.... It provides the hardware-backed security services such as the keymaster, gatekeeper, crypto, and so forth. These services are required for applications such as data storage encryption.

Reply:
done, thanks.